autoscheduler/scraper/models/section.py

```diff
@@ -15,10 +15,39 @@ class Section(models.Model):
     min_credits = models.IntegerField() # Will never be null
     max_credits = models.IntegerField(null=True) # Will be null in most cases
 
-    honors = models.BooleanField(null=True)
-    web = models.BooleanField(null=True)
+    honors = models.BooleanField(null=True, db_index=True)
+    web = models.BooleanField(null=True, db_index=True)
     # A course is asynchronous if none of its meetings have meeting times
-    asynchronous = models.BooleanField()
+    asynchronous = models.BooleanField(db_index=True)
+
+    # Normal instructional methods

```

Comment:
Sorry such a nit pick but could you order this to put all of the instructional methods last? So like move `max_enrollment` through `instructor` above this line, but `class Meta:` will still be below instructional methods.

Reply:
Yeah I thought about doing that and have no idea why I didn't, I think I just forgot tbh